doc/python/mapbox-county-choropleth.md

```diff
@@ -77,11 +73,11 @@ df = pd.read_csv("https://raw.githubusercontent.com/plotly/datasets/master/fips-
 df.head()
 ```
 
-### Choropleth map using plotly.express and carto base map (no token needed)
+### Choropleth map using plotly.express and carto base map
 
 [Plotly Express](/python/plotly-express/) is the easy-to-use, high-level interface to Plotly, which [operates on a variety of types of data](/python/px-arguments/) and produces [easy-to-style figures](/python/styling-plotly-express/).
 
-With `px.choropleth_mapbox`, each row of the DataFrame is represented as a region of the choropleth.
+With `px.choropleth_map`, each row of the DataFrame is represented as a region of the choropleth.
```

Comment:
if "DataFrame" is a class name, should it be in code font? If it's not, should it be lower case?